### PR TITLE
Henry

### PR DESCRIPTION
**Pull request by henryaudi**

### CODE REVIEW

File: src/pages/apply/ApplicationPage.tsx [hunk 1]
```diff
@@ -60,18 +60,22 @@ function ApplicationPage() {
...
-  const [courseId] = useState(1);
-  const [studentId] = useState(1);
+  const [studentId, setStudentId] = useState<number>(1);
+  const [courseId, setCourseId] = useState<number>(1);
+  const [taJobId, setTaJobId] = useState<number>(1);
   const [firstName, setFirstName] = useState('');
   const [lastName, setLastName] = useState('');
   const [courseName, setCourseName] = useState('');
   const [status, setStatus] = useState('');
-  const [hoursCanWorkPerWeek, setWeeklyHours] = useState('');
```
Comment (LonesomeTown) [inline src/pages/apply/ApplicationPage.tsx:70]: Plz keep consistency with the backend data structure.

File: src/pages/apply/ApplicationPage.tsx [hunk 2]
```diff
@@ -102,7 +106,7 @@ function ApplicationPage() {
       lastName,
       courseName,
       status,
-      hoursCanWorkPerWeek,
+      weeklyHours,
```
Comment (LonesomeTown) [inline src/pages/apply/ApplicationPage.tsx:109]: Plz keep consistency with the backend data structure.

**Comment by henryaudi** [comment]
Tony, please help me double check the naming in this PR, thanks! @LonesomeTown

**Review by LonesomeTown — CHANGES_REQUESTED** [review]

**Review by LonesomeTown — APPROVED** [review]
LGTM！

**Pull request merged**